scope.md

```diff
@@ -91,6 +91,51 @@ priority:
 * QR-code triggered payment. User points phone at QR code which represents a Web Page. In vanilla mode, Web Page includes "Buy" button to trigger Payment Request. But in streamlined mode, SPC transaction dialog is immediately displayed to the user for a default instrument associated with the payment method.
 * Tap-to-pay (NFC)
 
+## User Stories
+
+### Out of band enrollment
+
+* While mobile banking, Alice enroll her authenticator to be used with all her credit cards issued by this bank.
+* A few days later during checkout, Alice is prompted to confirm a transaction with one of those cards via her authenticator.
+
+### In-transaction enrollment, authentication same merchant
+
+* While checking out, Alice selects a payment instrument and is authenticated before being able to use it.
+* She is also prompted to enroll her authenticator with the bank in association with that instrument, to streamline future checkouts.
+* A few days later during checkout on the same merchant site, Alice is prompted to confirm a transaction with the same instrument by using the enrolled authenticator.
+
+### Authentication different merchant
+
+3) Having enrolled an authenticator previously (either out-of-band or during a transaction on any merchant site), Alice is shopping on an unrelated merchant
+site.
+* During checkout, Alice selects the same instrument and is prompted to authenticate by using the enrolled authenticator.
+
+### Enrollment for both payment authentication and account login
+
+* During a guest checkout experience, Alice provides information for payment
+and is given the option of creating a new user account with the merchant.
+* Alice is prompted to enroll her authenticator such that (1) it can be used to authenticate future payments with the instrument selected to make a payment and (2) it can be used to log into her new account with the merchant.
```

Comment:
I don't follow the implications of this case. This bullet point implies Alice enrolls once, with (it sounds like) `merchant1.com` as the relying party (since she is enrolling for login). But two bullet points later Alice is on (say) `merchant2.com` buying something and is able to user her authenticator - but with what credential? Did `merchant1.com` share the credential with `merchant2.com`, and if so then what is `merchant2.com` asserting here - and to whom?

For me, the case that is similar to this but makes more sense is if Alice enrolls her authenticator for login with *her bank*, and can then use the same credential for payment on `merchant.com`.

Reply:
I've updated the story and added a note that this may be more of a FIDO topic (sharing credentials) than an SPC topic, but that we've included it for discussion.